### DISCUSSION TITLE
Welcome to react-music-player Discussions!

### DISCUSSION THREAD

**Discussion by lijinke666** [lijinke666/react-music-player#231] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zzikkzzakk** [comment]
@lijinke666 : Really great music player! I'm just starting to integrate it into my website, but already ❤️ it. 
I will share the result as soon as it's ready 
Great work! 🎸😎

**Comment by dsomel21** [comment]
Thanks for making such a great tool!
